### DISCUSSION TITLE
swiper

### DISCUSSION THREAD

**Discussion by beixiangsiyu** [ant-design/ant-design-mobile#5788] [Q&A]
when use swiper,it can autoplay,but cant touchmove

**Comment by miracles1919** [comment]
Are you set `allowTouchMove=false` ?

**Reply by beixiangsiyu** [reply]
sorry, i use default settings

**Comment by beixiangsiyu** [comment]
![image](https://user-images.githubusercontent.com/74031376/199879786-da4538fe-aaaa-4a82-9b19-7c706e21d128.png)

**Reply by miracles1919** [reply]
Count you provide a online demo by [codesandbox](https://codesandbox.io/s/antd-mobile-snrxr?file=/package.json) or [stackblitz](https://stackblitz.com/edit/antd-mobile?file=index.tsx)?

**Reply by beixiangsiyu** [reply]
https://codesandbox.io/s/dazzling-sun-3235ko
